product_pack/models/product_pack_line.py

```diff
@@ -54,6 +54,14 @@ def _check_recursion(self):
                     )
                 pack_lines = pack_lines.mapped("product_id.pack_line_ids")
 
-    def get_price(self):
+    def get_price(self, price_type, currency=False, company=False, date=False):
         self.ensure_one()
+        if price_type == "standard_price":
+            price_currency = self.product_id.cost_currency_id
+            company = company or self.env.company
+            date = date or fields.Date.context_today(self)
+            price = self.product_id["standard_price"]
```

Comment:
```suggestion
            price = self.product_id.standard_price
```

Reply:
Yes, totally! thats why i made a new PR! Thanks for reviewed it